public/pages/Dashboard/components/DashboardEmptyPrompt/DashboardEmptyPrompt.js

```diff
@@ -17,8 +18,11 @@ const createTriggerText =
 const editTriggerConditionsText =
   'There are no existing alerts. Adjust trigger conditions to start alerting. Once an alarm is triggered, the state will show in this table.';
 
-const createMonitorButton = (
-  <EuiSmallButton fill href={`${PLUGIN_NAME}#${APP_PATH.CREATE_MONITOR}`}>
+const getCreateMonitorButton = () => (
```

Comment:
do we need iconType in this?

Reply:
I will add it in follow up PR where I also fix some issues with the multi data source setup